compiler/bytecode/instruction.go

```diff
@@ -157,7 +157,7 @@ func (as *ArgSet) Names() []string {
 	return as.names
 }
 
-func (as *ArgSet) FindIndex(name string) int {
+func (as *ArgSet) findIndex(name string) int {
```

Comment:
If changing this function name didn't break anything, maybe we can remove it?

Reply:
Good catch!

Reply:
Removed `findIndex()` on https://github.com/goby-lang/goby/pull/817